src/main/java/boot/project/demo/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package boot.project.demo.service.impl;
+
+import boot.project.demo.service.FileReaderService;
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String[]> readFile(String filePath) {
```

Comment:
usually it's more complicated to work with List<String[]> so you cant try to read your file into List<String> or read record into model/dto right away